compose/galaxy-web/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup && \
     # apt-get update -qq && apt-get upgrade -y && \
     # apt-get update -qq && \
     apt-get install --no-install-recommends -y \
-    nginx-extras nginx-common supervisor autofs libslurm32 libslurmdb32 && \
```

Comment:
Although beyond this PR, it would be really great if things like slurm dependencies could be installable via ansible and then that could be enabled or disabled via injection of arguments on build time.

Reply:
@pcm32 are you using the pre-build images or are you building the images on your own again? Or what is your preferred use-case? We could also define those libs via an ENV and create different web containers during build-time - not sure this is a use-case.

Reply:
@bgruening I'm probably still building my own because I need to point to a version of ansible galaxy extras that includes this: https://github.com/galaxyproject/ansible-galaxy-extras/pull/213
On my setup, it fails because uswgi process I think points to that galaxy.yml but that file doesn't exists at that point... the test not passed seems to be unrelated to the logic added on that PR.

Reply:
@bgruening in your compose setup works without my PR ansible-galaxy-extras due to https://github.com/bgruening/docker-galaxy-stable/blob/dev/compose/docker-compose.yml#L327 but the current PR (464) is actually removing that line, so the compose setup will need the solution proposed in https://github.com/bgruening/docker-galaxy-stable/blob/dev/compose/docker-compose.yml#L327 I think....